app.rb

```diff
@@ -0,0 +1,101 @@
+require "sinatra/base"
+require "sinatra/flash"
+require "./lib/user.rb"
+require "./lib/message.rb"
+require_relative "./database_connection_setup.rb"
+
+class Chitter < Sinatra::Base
```

Comment:
Your controller this week looks much improved over your RPS implementation. It's great to see you using RESTful routing conventions and keeping your routes concise and free of logical heavy lifting.